shared/common-adapters/usernames.js

```diff
@@ -6,13 +6,13 @@ import {isMobile} from '../constants/platform'
 
 import type {Props} from './usernames'
 
-function usernameText ({type, users, style, inline, redColor, backgroundMode, colorFollowing}: Props) {
+function usernameText ({type, users, style, inline, redColor, backgroundMode, colorFollowing, colorBroken = true}: Props) {
```

Comment:
i originally was using it but then later opted to not pass the broken info, then didn't remove it, but its very simple in its logic and we already had a flag to control colorFollowing so i left it in the api of the common component